### ISSUE TITLE
minor issue

### ISSUE THREAD

**Issue by shaoyuyoung** [ise-uiuc/WhiteFox#7]
requirement file (https://github.com/ise-uiuc/WhiteFox/blob/main/requirement.txt) should be renamed `requirements.txt` as the command in readme file is `pip install -r requirements.txt`

**Comment by shaoyuyoung** [comment]
Moreover, here needs to be modified  as `--output-dir`
![image](https://github.com/user-attachments/assets/bdefa04f-3fc8-4bfc-9e25-407cb8ecc8d5)

**Comment by YangChenyuan** [comment]
Thanks so much for pointing them out! They are fixed in #8

**Comment by shaoyuyoung** [comment]
BTW, I have a small question about whitefox.

I have reproduced the experiment (`torch inductor`). I found that the number of generated tests (`tested.log`) is **610** and the number of success tests (`success.log`) is **320**.  

It seems that the success rate of pass is only **52.46% (320/610)**. That seems a little low, right?

theoretically speaking, the rest tests should be `bugs` which Whitefox detects. However, I have manually checked these failed tests and found that the majority of them (approximately 80%) are **false alerts**. This means that the crash was caused by LLM generating invalid tests.

I am wondering whether this is because of the limited capability of `ise-uiuc/Magicoder-S-DS-6.7B`?
